.github/workflows/hugo.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      HUGO_VERSION: 0.108.0
+      HUGO_VERSION: 0.119.0
```

Comment:
Czy mógłbyś zrobić oddzielnego pull requesta z zamym podbiciem wersji hugo?

Teraz jeśli zmergujemy i coś przestanie działać to nie będziemy pewni, czy  zawiniła tu wersja HUGO, czy też inne zmiany dotyczące np konfigów.